executor/executor.cc

```diff
@@ -818,7 +815,7 @@ void execute_one()
 		memset(&call_props, 0, sizeof(call_props));
 	}
 
-	if (!colliding && !collide && running > 0) {
+	if (running > 0) {
```

Comment:
This subtly changes behavior. Previously we did not wait for syscalls in collide mode. Now we will wait for them.
Wanted to double check that this is an intentional change.
I think it's fine.

Reply:
Yes, this was an intentional change as we still would like to print some call output info back to the syz-fuzzer.